modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/nw-nodes-peer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-egress_{context}"]
+= Egress traffic control for AdminNetworkPolicy and BaselineAdminNetworkPolicy
+
+As a cluster administrator you can use `AdminNetworkPolicy` (ANP) resource in conjunction with `BaselineAdminNetworkPolicy` (BANP) resource to secure and limit egress traffic. Using these namespace scoped policies together you can do the following:
+
+* Create egress traffic control for cluster nodes.
+
+* Apply egress traffic control towards Kubernetes API servers.
+
+* Implement egress traffic control towards external destinations.
+
+//For example, as a cluster administrator, you can enforce security for northbound traffic by restricting traffic from specific cluster workloads to all or specific destinations outside the cluster. Or, you can setup policies that grant access to your cluster-external DNS server even if namespace admins create `NetworkPolicy` resources that block cluster-external egress.
+
+[id="egress-traffic-control-towards-cluster-nodes_{context}"]
+== Using egress traffic control for ANP and BANP
+
+Using the `matchLabels` and `matchExpressions` fields, cluster administrators can select nodes in a cluster based on `nodes` and `networks` labels and apply egress traffic policy to these nodes. The following are benefits to using label selectors when blocking access from cluster workloads to specific ports on cluster nodes:
```

Comment:
matchLabels and/or matchExpressions are for nodes. networks have the multiple CIDRs

Reply:
I tried to separate the distinctions more. Let me know if I've got something wrong.